<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoAdminDialog</class>
 <widget class="QDialog" name="QWoAdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>This password is very important and must be remembered.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pass">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>22</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPassReset">
          <property name="text">
           <string>save</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../private/skins/black/save.png</normaloff>../private/skins/black/save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkStartup">
          <property name="text">
           <string>Login when startup.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkPassword">
          <property name="text">
           <string>Look up password.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkTunnelOpen">
          <property name="text">
           <string>Allow connections from the local area network.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkLocalshell">
          <property name="text">
           <string>Local shell tab auto start on app start.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSerialport">
          <property name="text">
           <string>Serialport tab auto start on app start.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkTunnelDaemon">
          <property name="text">
           <string>Set tunnel function run in daemon mode.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkGroupExpand">
          <property name="text">
           <string>Allow session group auto restore last expanded state.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkOpacity">
          <property name="text">
           <string>Allow to set main window opacity.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSshAgent">
          <property name="text">
           <string>Allow to use the external identity files such as ssh-agent or pageant.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
